tests/clients/test_user.py

```diff
@@ -11,68 +11,118 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 import uuid
+from datetime import datetime
 
 import pytest
 from marshmallow import ValidationError
+from dateutil.tz import UTC
 
-from sherlockml.clients.user import (
-    UserClient,
-    User,
-    UserSchema,
-    AuthenticationResponse,
-    AuthenticationResponseSchema,
-)
-
+from sherlockml.clients.user import UserClient, User, UserSchema, GlobalRole
 from tests.clients.fixtures import PROFILE
 
 USER_ID = uuid.uuid4()
+CREATED_AT = datetime(2018, 3, 10, 11, 32, 6, 247000, tzinfo=UTC)
+CREATED_AT_STRING = "2018-03-10T11:32:06.247Z"
+
+TEST_USER_JSON = dict(
```

Comment:
I'd prefer we use a dictionary literal rather than calling `dict` (using a literal is a bit faster, as the interpreter doesn't need to lookup `dict`).